canopen/objectdictionary/__init__.py

```diff
@@ -387,7 +395,7 @@ def decode_raw(self, data: bytes) -> Union[int, float, str, bytes, bytearray]:
             return data.rstrip(b"\x00").decode("ascii", errors="ignore")
```

Comment:
Why not switch around the stripping and decoding here as well, for consistency?

Reply:
I think order doesn't matter, as "ascii" doesn't encode `\x00`

Reply:
What I meant was `data.rstrip(b"\x00").decode("ascii") == data.decode("ascii").rstrip("\x00")`

Reply:
That's clear.  It just seems clearer if the order of actions is the same as for the 16-bit case.